src/object-properties.js

```diff
@@ -175,7 +175,7 @@ const properties = {
  * @property {object=} url.qStaticContentUrlDef
  * @property {string=} url.qStaticContentUrlDef.qUrl - URL represented as a string
  * @property {('topLeft'|'centerLeft'|'bottomLeft'|'topCenter'|'centerCenter'|'bottomCenter'|'topRight'|'centerRight'|'bottomRight'|'top-left'|'center-left'|'bottom-left'|'top-center'|'center-center'|'top-right'|'center-right'|'bottom-right')} [position='center-center'] - Image position
- * @property {('auto'|'alwaysFit'|'fitWidth'|'fitWidth'|'fitHeight'|'fill'|'alwaysFill')=} [size='auto'] - Size of the image, relative to the button
+ * @property {('auto'|'alwaysFit'|'fitWidth'|'fitWidth'|'fitHeight'|'fill'|'alwaysFill'|'stretchFit')=} [size='auto'] - Size of the image, relative to the button
```

Comment:
there are two fitWidth here
```suggestion
 * @property {('auto'|'alwaysFit'|'fitWidth'|'fitHeight'|'fill'|'alwaysFill'|'stretchFit')=} [size='auto'] - Size of the image, relative to the button
```